bugs.txt

```diff
@@ -343,3 +347,7 @@ bpo-8674:
     author: Tomas Hoger
     date: 2010-05-10 13:43:22
     title: 'audioop: incorrect integer overflow checks'
+gh-88715:
+    author: 7320040f-9856-4955-8493-c95773d76b91
```

Comment:
For issues migrated from bpo to gh, author information may be lost. Maybe I need to update them manually. 

In addition, I tested that the author information of the recently submitted issues on github can be correctly obtained through github api.

Reply:
> author: 7320040f-9856-4955-8493-c95773d76b91

Oh. I would prefer an user name rather than a UUID. Ok, maybe for that one it's better to use the bpo number. Maybe it's the "mannequin" issue.